Nez.Portable/Assets/Tiled/Runtime/TiledMapLoader.cs

```diff
@@ -75,7 +75,7 @@ public static TmxMap LoadTmxMap(this TmxMap map, XDocument xDoc)
 		private static void UpdateMaxTileSizes(TmxTileset tileset)
```

Comment:
none of this is related to the hardware mode switch...

Reply:
No that's actually the previous PR I made about adding template file support to the tiledmaploader. I'm not sure why that's being included in this PR.